### PR TITLE
Absolute Imports in Tracking

### PR DESCRIPTION
**Pull request by theaverageguy**
Fixes Issue #1001

### CODE REVIEW

**Comment by arokem** [comment]
A good way to sort out this kind of thing is to test it on your own machine before pushing. You can run the tests locally with nose.

**Comment by sahmed95** [comment]
In https://github.com/nipy/dipy/pull/1009/commits/927dcc646826af3409f8ba07d521768628b06d2f#diff-dabbfc9d5ab28faed60ef453735df58dR10

I guess you missed dipy.reconst.cache

File: dipy/tracking/utils.py [hunk 1]
```diff
@@ -65,7 +65,7 @@
 from dipy.tracking import metrics
 
 # Import helper functions shared with vox2track
-from ._utils import (_mapping_to_voxel, _to_voxel_coordinates)
+from tracking._utils import (_mapping_to_voxel, _to_voxel_coordinates)
```
Comment (arokem) [inline dipy/tracking/utils.py]: I believe this needs to be:
    
    from dipy.tracking._utils ...

**Comment by arokem** [comment]
Any chance to get this rebased on master and to make that small fix?

**Comment by sahmed95** [comment]
Could make it myself, but is there a way for two people to push commits to the same PR ? (Without making a new branch/pull request). Just out of curiosity - can multiple people work on the same PR ?

**Comment by arokem** [comment]
Only if you are made a collaborator on that fork. It would probably result
in a mess, though.

You can make a PR on the fork, against this branch.

**Comment by sahmed95** [comment]
Will look it up. Thanks. But I guess it will be messy.

**Comment by theaverageguy** [comment]
I have my finals going on for about two weeks. Sorry for no response. I am
fixing everything today after 4ish IST.

**Comment by arokem** [comment]
This one seems abandoned. Closing, but if someone wants to pick it back up, feel free to reopen.

**Pull request closed without merging**